app/form_objects/forms/payment_link_form.rb

```diff
@@ -3,9 +3,8 @@
 class Forms::PaymentLinkForm < BaseForm
   attr_accessor :form, :payment_url
 
-  PAY_URL_REGEX = /\Ahttps:\/\/www\.gov\.uk\/payments\/.+\/.+/
-
-  validates :payment_url, url: true, format: { with: PAY_URL_REGEX, message: :url }, if: -> { payment_url.present? }
+  validates :payment_url, url: true, if: -> { payment_url.present? }
+  validate :is_pay_url?, if: -> { payment_url.present? }
```

Comment:
should it be `validates`? also think this change should be in the earlier commit

Reply:
I believe `validate` is correct here - I think:

- `validate` is for [custom methods](https://guides.rubyonrails.org/active_record_validations.html#custom-methods) (like `is_pay_url?` here), 
- `validates` is for the builting [validation helpers](https://guides.rubyonrails.org/active_record_validations.html#validation-helpers) (like `url: true` on the line above)
- `validates_with` is for [custom validator classes](https://guides.rubyonrails.org/active_record_validations.html#custom-validators)

Reply:
I've moved where this is comitted, but I'll leave the `validate` as it's a custom validation